server/src/models/validation/query-param.js

```diff
@@ -4,9 +4,10 @@ const Joi = require('@hapi/joi');
 module.exports = {
 	limit: Joi.number().integer().min(1).max(20),
 	offset: Joi.number().integer().min(0).max(5000),
-	search: Joi.string(), // hmmm
+	search: Joi.string().allow(''), // Is there anywhere that this is used that shouldn't allow empty strings?
 	expand: Joi.string().regex(/[a-zA-Z,]*/),
 	priority: Joi.boolean(),
 	status: Joi.string().regex(/[0-9,]*/), // should probably be moved out to somewhere else? change to statusIn?
 	playerIDs: Joi.string().regex(/[0-9,]*/),
+	order: Joi.string().valid("date", "time").required(), // Should this be moved somewhere else? Should it accept more than just date and time?
```

Comment:
Use single apostrophe for strings:
```suggestion
	order: Joi.string().valid('date', 'time').required(), // Should this be moved somewhere else? Should it accept more than just date and time?
```

Reply:
Fixed in b1bd918